app/src/main/java/com/pokerio/app/utils/GameState.kt

```diff
@@ -500,8 +500,8 @@ object GameState {
 
     @OptIn(ExperimentalSerializationApi::class)
     fun startGame(card1: String, card2: String, playersString: String) {
-        gameCard1 = GameCard(card1.slice(0..1), card1.slice(2..2))
-        gameCard2 = GameCard(card2.slice(0..1), card2.slice(2..2))
+        gameCard1 = GameCard.fromString(card1)
```

Comment:
GameCard.fromStrings throws IllegalArgumentException, shouldn't we catch and handle it?

Reply:
Done